### PR TITLE
Add wercker (cli) Latest

### PR DESCRIPTION
**Pull request by xakraz**
The CLI utility of the CI Saas http://wercker.com/

### CODE REVIEW

File: Casks/wercker-cli.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask :v1 => 'wercker-cli' do
+  version ':lastest'
+  sha256 '3db5ec63989170414293af1c07fd83ec86d1d14c6b5413d2604f97ac771a5294'
+
+  # The vendor use Amazon S3 storage service to deliver files over HTTP
```
Comment (vitorgalvao) [inline Casks/wercker-cli.rb]: Comment is wrong. It must be Needs a `# amazonaws.com is the official download host per the vendor homepage` ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

File: Casks/wercker-cli.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  url 'https://s3.amazonaws.com/downloads.wercker.com/cli/stable/darwin_amd64/wercker'
+  name 'wercker'
+  name 'wercker-cli'
+  homepage 'http://wercker.com/'
+  license :unknown
+
+  container :type => :naked
```
Comment (vitorgalvao) [inline Casks/wercker-cli.rb]: Are you sure this is needed?

File: Casks/wercker-cli.rb [hunk 3]
```diff
@@ -0,0 +1,18 @@
+cask :v1 => 'wercker-cli' do
+  version ':lastest'
+  sha256 '3db5ec63989170414293af1c07fd83ec86d1d14c6b5413d2604f97ac771a5294'
+
+  # The vendor use Amazon S3 storage service to deliver files over HTTP
+  url 'https://s3.amazonaws.com/downloads.wercker.com/cli/stable/darwin_amd64/wercker'
+  name 'wercker'
+  name 'wercker-cli'
+  homepage 'http://wercker.com/'
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/wercker-cli.rb]: Can’t you figure out the license?

Comment (lumaxis) [inline Casks/wercker-cli.rb]: The [license](https://github.com/wercker/wercker-cli/blob/master/LICENSE) is MIT

Comment (vitorgalvao) [inline Casks/wercker-cli.rb]: `license :mit`, then.

**Comment by adityadalal924** [comment]
should be

```
version: latest
```

File: Casks/wercker-cli.rb (hunk 1 above)
Comment (xakraz) [inline Casks/wercker-cli.rb]: @vitorgalvao <https://github.com/vitorgalvao>, OK I thought it was an
"example".

I have updated the comment.

**Comment by vitorgalvao** [comment]
Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/master/Casks/acorn.rb).

File: Casks/wercker-cli.rb [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  sha256 '3db5ec63989170414293af1c07fd83ec86d1d14c6b5413d2604f97ac771a5294'
+
+  # amazonaws.com is the official download host per the vendor homepage
+  url 'https://s3.amazonaws.com/downloads.wercker.com/cli/stable/darwin_amd64/wercker'
+  name 'wercker'
+  name 'wercker-cli'
+  homepage 'http://wercker.com/'
+  license :mit
+
+  container :type => :naked
```
Comment (vitorgalvao) [inline Casks/wercker-cli.rb]: Have you confirmed the cask doesn’t work without this line?

File: Casks/wercker-cli.rb (hunk 3 above)
Comment (xakraz) [inline Casks/wercker-cli.rb]: You are right, I assumed (wrongly) that it was closed sources ...

After your comment I saw https://github.com/wercker/wercker-cli

File: Casks/wercker-cli.rb (hunk 2 above)
Comment (xakraz) [inline Casks/wercker-cli.rb]: Do you mean, am I sure that the recipe is needed compared to:
* the way mentioned here : curl -L https://install.wercker.com | sh
* and the other way mentioned here : http://wercker.com/downloads/


Ok, it is not really needed, but what is the purpose to Brew cask then ?
By the way, I founded a ```wercker``` recipe for the Desktop GUI version, I
was expecting the same for the CLI.

Do you think I should rather to a Brew recipe based on
https://github.com/wercker/wercker-cli ?

File: Casks/wercker-cli.rb (hunk 4 above)
Comment (xakraz) [inline Casks/wercker-cli.rb]: Yes, I confirm that without the line

```

container :type => :naked

```

The cask fails

**Comment by vitorgalvao** [comment]
Just noticed. Can’t this be instead expand the already present [`wercker`](https://github.com/caskroom/homebrew-cask/blob/master/Casks/wercker.rb) cask?.

**Comment by xakraz** [comment]
Is it possible to have multiple URL stanza in a recipe ?

By the way, as a user, I prefer to be able to download the CLI alone rather
than the bundled Desktop App if possible, but everything is possible.

**Comment by vitorgalvao** [comment]
> Is it possible to have multiple URL stanza in a recipe?

No.

> By the way, as a user, I prefer to be able to download the CLI alone rather than the bundled Desktop App if possible, but everything is possible.

https://github.com/caskroom/homebrew-cask/issues/2992.

File: Casks/wercker-cli.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+cask :v1 => 'wercker-cli' do
+  version :latest
+  sha256 :no_check
+
+  # amazonaws.com is the official download host per the vendor homepage
+  url 'https://s3.amazonaws.com/downloads.wercker.com/cli/stable/darwin_amd64/wercker'
```
Comment (vitorgalvao) [inline Casks/wercker-cli.rb]: Can you point me to where the link is, so I can confirm it?

Comment (xakraz) [inline Casks/wercker-cli.rb]: The link for Wercker Desktop version ?

https://github.com/caskroom/homebrew-cask/blob/master/Casks/wercker.rb#L5

Comment (vitorgalvao) [inline Casks/wercker-cli.rb]: No. Where on the website is this particular download link?

Comment (xakraz) [inline Casks/wercker-cli.rb]: http://wercker.com/downloads/

Pick your OS and install the CLI

```
curl -L https://s3.amazonaws.com/downloads.wercker.com/cli/stable/darwin_amd64/wercker -o /usr/local/bin/wercker
```

**Comment by vitorgalvao** [comment]
Everything looks fine. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)? Please @mention me after you do, so I get an email to check back.

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It needed some fixes, so they were made in 7173d8fd823dc2cb4f445217bf010b9879a1922e. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Comment by xakraz** [comment]
@vitorgalvao Thanks !

**Pull request closed without merging**